ocs_ci/ocs/fio_artefacts.py

```diff
@@ -0,0 +1,200 @@
+import configparser
+import logging
+import textwrap
+import yaml
+
+from ocs_ci.ocs import constants, ocp
+from ocs_ci.utility.utils import (
+    config_to_string,
+    update_container_with_mirrored_image
+)
+
+log = logging.getLogger(__name__)
+
+
+def get_fs_conf():
+    """
+    Basic fio configuration for upgrade utilization for fs based pvcs
+
+    Returns:
+        str: fio configuration
+
+    """
+    # TODO(fbalak): handle better fio size
+    fio_size = 1
+    return textwrap.dedent(f"""
+        [readwrite]
+        readwrite=randrw
+        buffered=1
+        blocksize=4k
+        ioengine=libaio
+        directory=/mnt/target
+        size={fio_size}G
+        time_based
+        runtime=24h
+        numjobs=10
+        """)
+
+
+def get_block_conf():
+    """
+    Basic fio configuration for upgrade utilization for block based pvcs
+
+    Returns:
+        str: fio configuration
+
+    """
+    # TODO(fbalak): handle better fio size
+    fio_size = 1
+    return textwrap.dedent(f"""
+        [readwrite]
+        readwrite=randrw
+        buffered=1
+        blocksize=4k
+        ioengine=libaio
+        filename=/dev/rbdblock
+        size={fio_size}G
+        time_based
+        runtime=24h
+        numjobs=10
+        """)
```

Comment:
I suggest to keep this fio config (`get_fs_conf` and `get_block_conf`)  in the test where it was taken from (file `tests/ecosystem/upgrade/conftest.py`). Note that you don't have to keep fio config in a fixture.

Reply:
Ok, `get_fs_conf` and `get_block_conf` were taken out.